test/Game-test.js

```diff
@@ -8,13 +8,13 @@ import Player from '../src/Player';
 const expect = chai.expect;
 chai.use(spies);
 
-let game;
+let game = new Game(data, ['Kayla', 'Brady', 'Allison']);
 
 describe('Game', () => {
   
-  beforeEach(() => {
-    game = new Game(data, ['Kayla','Brady','Allison']);
-  });
+  // beforeEach(() => {
+  //   game = new Game(data, ['Kayla','Brady','Allison']);
+  // });
```

Comment:
Remove debugging statements/commented out code like this before submitting a PR -- a good way to avoid committing these is by doing a `git add --patch` rather than just a `git add .` Patch will allow you to walk through all of your changes and manually choose if you want them to be a part of your commit or not